verifier/src/lib.rs

```diff
@@ -72,7 +72,7 @@ impl fmt::Display for VerificationError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         use VerificationError::*;
         match self {
-            VerifierError(e) => write!(f, "there was a verification error: {e}"),
+            VerifierError(e) => write!(f, "{e}"),
```

Comment:
optional: shouldn't we qualify the domain of the error? That in the sense of differentiating between the other variants of this same error, as opposed to describe the verification error directly (so it becomes easier for the user to find the source of the error).

Reply:
The `VerifierError` message from Winterfell should contain enough detail here - but we can revisit later.